src/lib/routers/history.ts

```diff
@@ -119,7 +120,8 @@ class BrowserHistory implements Router {
   /**
    * Pushes a search state into the URL.
    */
-  public write(routeState: RouteState): void {
+  public write(routeState: RouteState, replace: boolean = false): void {
```

Comment:
Correct me if I'm wrong, adding an optional parameter should not be a breaking change for someone relying on this type.

Reply:
yes, that's not an issue